centreon/src/Core/HostGroup/Infrastructure/API/DuplicateHostGroups/DuplicateHostGroupsRoute.yaml

```
@@ -0,0 +1,5 @@
+DuplicateHostGroups:
+  methods: POST
+  path: /configuration/hosts/groups/_duplicate
+  controller: 'Core\HostGroup\Infrastructure\API\DuplicateHostGroups\DuplicateHostGroupsController'
+  condition: "request.attributes.get('version') >= 25.03"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)